src/main/java/org/broadinstitute/hellbender/utils/variant/GATKVCFConstants.java

```diff
@@ -173,6 +172,8 @@ their names (or descriptions) depend on some threshold.  Those filters are not i
     public final static String STRICT_STRAND_BIAS_FILTER_NAME =               "strict_strand";
     public final static String N_RATIO_FILTER_NAME =                           "n_ratio";
     public final static String CHIMERIC_ORIGINAL_ALIGNMENT_FILTER_NAME =       "numt_chimera"; //mitochondria
+    public final static String ALLELE_FRACTION_FILTER_NAME =                   "low_allele_fraction";
```

Comment:
For beautiful formatting of `FilterMutectCalls`' filter analysis output, could you makes the filter names more than 8 and less than 16 characters?